.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Release
+
+on:
+  push:
+    branches: [master, next]
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PUBLIC_CDN_URL: 'https://embed.typeform.com'
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  init:
+    name: Init
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.ci-skip-step.outputs.ci-skip }}
+      skip-not: ${{ steps.ci-skip-step.outputs.ci-skip-not }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: mstachniuk/ci-skip@master
+        id: ci-skip-step
+        with:
+          commit-filter: '[skip ci]'
+  release_npm:
+    name: Release to NPM
+    runs-on: ubuntu-latest
+    needs: init
+    if: ${{ needs.init.outputs.skip == 'false' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # Use custom token from repo secrets to allow semantic release to push commit:
+          # https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md#pushing-packagejson-changes-to-a-master-branch
+          persist-credentials: false
+          token: ${{ secrets.GH_TOKEN }}
+      - uses: actions/setup-node@v1
+      - uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
+            ~/.cache
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - run: yarn install
+      - run: yarn build
```

Comment:
Maybe we should run the build via lerna to build each package?

Reply:
So, replace it with `yarn lerna bootstrap`?

Reply:
Like this maybe?
```
yarn lerna bootstrap
yarn lerna run build
```

PR workflow does  `yarn install` only. Maybe it should be updated too.